.github/ISSUE_TEMPLATE/question.md

```diff
@@ -8,4 +8,4 @@ assignees: ""
 
 Please read the following before posting:
 
-The Enterprise Search team at Elastic maintains this library and are happy to help. Try posting your question to the [Elastic Enterprise Search](https://discuss.elastic.co/c/enterprise-search/84) discuss forums. Be sure to mention that you're using search-ui and also let us know what backend your using; whether it's App Search, Site Search, Workplace Search, Elasticsearch, or something else entirely.
+The Enterprise Search team at Elastic maintains this library and are happy to help. Try posting your question to the [Elastic Enterprise Search](https://discuss.elastic.co/c/enterprise-search/84) discuss forums. Be sure to mention that you're using Search UI and also let us know what backend your using; whether it's App Search, Site Search, Workplace Search, Elasticsearch, or something else entirely.
```

Comment:
Everywhere in this PR I use "Search UI" to refer to the whole library and "search-ui" to refer to the `search-ui` core package.